Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,60 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "without-test", "Skip running build-time tests (not recommended)"
+  depends_on "cmake" => :run
+  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
+  depends_on "robodoc" => [:recommended, :build]
+  depends_on :fortran
+
+  def install
+    mktemp do
+      args = ["-DUSE_GNU_INSTALL_CONVENTION:BOOL=TRUE"]
+      args << "-DENABLE_UNICODE:BOOL=TRUE" if build.with? "unicode-support"
+      args << "-DSKIP_DOC_GEN:BOOL=TRUE" if build.without? "robodoc"
+      system "cmake", buildpath, *(args + std_cmake_args)
+      system "make"
```

Comment:
a direct `make install` works, but `make test` does not work unless `make` is run before hand. 

Would you consider that a bug in json-fortran, to be solved upstream?

As a temporary workaround I can do this:
```ruby
if build.with? “test”
  system “make”
  system “make”, “test”
end
```